config/config.go

```diff
@@ -206,6 +206,10 @@ type BaseConfig struct { //nolint: maligned
 	// Output format: 'plain' (colored text) or 'json'
 	LogFormat string `mapstructure:"log-format"`
 
+	// The max length of the log message and the log field value:
+	// `0` is the default value (no length limit)
```

Comment:
we should be clear about what kind of unit we're using here. (bytes, characters, number of k/v pairs in structured messages, etc.) 

how are negatives handled?

Reply:
the value validation is in here:
https://github.com/tendermint/tendermint/pull/7258/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R329
If the user puts a negative value, the node will return an error immediately.